### DISCUSSION TITLE
Model Viewer not displaying model in AR on chrome latest version on android

### DISCUSSION THREAD

**Discussion by GamerDevarsh** [google/model-viewer#4607] [Q&A]
model viewer is displaying my GLB models in AR on safari just fine, but the AR viewer hangs and crashes (even when i try to see the examples given on the model viewer site) on latest chrome version on android (last updated on 11 Dec), the prior version of chrome is displaying the model just fine. It's showing "AR NOT TRACKING" and AR Viewer crashes. Please help me with the issue resolution.

**Comment by elalish** [comment]
Oh, that's bad - I have a repro on my Pixel 7 too.

**Reply by elalish** [reply]
Hmm, strange - it only happened for me on my first WebXR session - after closing and reopening it's working now. And I haven't seen it crash yet. Is the behavior consistent for you?
